testsuite/performance/tests/pom.xml

```diff
@@ -48,11 +48,25 @@
         <keycloak.ds.max-pool-size>100</keycloak.ds.max-pool-size>
         <keycloak.ds.pool-prefill>true</keycloak.ds.pool-prefill>
         <keycloak.ds.ps-cache-size>100</keycloak.ds.ps-cache-size>
-        
+
         <keycloak-lb.jvm.memory>-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m</keycloak-lb.jvm.memory>
         <keycloak-lb.http.max-connections>500</keycloak-lb.http.max-connections>
         <keycloak-lb.worker.io-threads>2</keycloak-lb.worker.io-threads>
         <keycloak-lb.worker.task-max-threads>16</keycloak-lb.worker.task-max-threads>
+
+        <docker.db.cpuset>0-1</docker.db.cpuset>
+        <docker.keycloak.cpuset>2-3</docker.keycloak.cpuset>
+        <docker.keycloak.dc1.cpuset>2</docker.keycloak.dc1.cpuset>
+        <docker.keycloak.dc2.cpuset>3</docker.keycloak.dc2.cpuset>
+        <docker.monitoring.cpuset>0</docker.monitoring.cpuset>
+
+        <docker.db.memlimit>2g</docker.db.memlimit>
+        <docker.keycloak.memlimit>2g</docker.keycloak.memlimit>
+
+        <keycloak.worker.io.threads>2</keycloak.worker.io.threads>
```

Comment:
The following 4 properties are already defined higher in the file. There is `keycloak.jvm.memory` instead of `keycloak.memory.settings`.

Reply:
Have missed that, thanks.